tests/Fixtures/app/config/config_common.yml

```diff
@@ -58,25 +58,25 @@ api_platform:
         nesting_separator: __
     name_converter:                    'app.name_converter'
     enable_fos_user:                   true
-    collection:
-        order_parameter_name:          'order'
-        order:                         'ASC'
-        pagination:
-            client_enabled:            true
-            client_items_per_page:     true
-            client_partial:            true
-            items_per_page:            3
     exception_to_status:
         Symfony\Component\Serializer\Exception\ExceptionInterface: !php/const Symfony\Component\HttpFoundation\Response::HTTP_BAD_REQUEST
         ApiPlatform\Core\Exception\InvalidArgumentException: !php/const Symfony\Component\HttpFoundation\Response::HTTP_BAD_REQUEST
         ApiPlatform\Core\Exception\FilterValidationException: !php/const Symfony\Component\HttpFoundation\Response::HTTP_BAD_REQUEST
     http_cache:
         invalidation:
             enabled: true
-        max_age:        60
-        shared_max_age: 3600
-        vary:           ['Accept', 'Cookie']
-        public:         true
+    defaults:
+        pagination_client_enabled:            true
+        pagination_client_items_per_page:     true
+        pagination_client_partial:            true
+        pagination_items_per_page:            3
+        order_parameter_name:          'order'
```

Comment:
There's no such key supported in the defaults.

Reply:
:+1: in fact the defaults are meant to be used on the `@ApiResource` (IIUC) and parameter names are supposed to be globally defined ?

Reply:
Parameter names are customizable per-filter, so indeed they have no place in the resource / operation attributes, I think...